crates/uv-toolchain/src/downloads.rs

```diff
@@ -50,6 +51,11 @@ pub enum Error {
     },
     #[error("Failed to parse toolchain directory name: {0}")]
     NameError(String),
+    #[error("Cannot download toolchain for request: {0}")]
+    InvalidRequestKind(ToolchainRequest),
+    // TODO(zanieb): Implement display for `PythonDownloadRequest`
```

Comment:
It looks like all the individual parts implement display. Consider just knocking it out in this PR?